src/components/buttons/Delete.tsx

```diff
@@ -62,9 +68,28 @@ export const DeleteButton = ({ entityName, entityTitle, callback }: IProps) => {
             <Button ref={cancelRef} onClick={onClose}>
               No
             </Button>
-            <Button colorScheme='red' ml={3} onClick={handleConfirmClick}>
-              Yes
-            </Button>
+            {!isDisabled && (
```

Comment:
I think we can do this in a more DRY way - let's jump on a call

Reply:
Have pushed what we added :)